src/components/resumption/src/last_state_impl.cc

```diff
@@ -69,8 +70,9 @@ Json::Value& LastStateImpl::get_dictionary() {
 void LastStateImpl::LoadStateFromFileSystem() {
   std::string buffer;
   bool result = file_system::ReadFile(app_info_storage_, buffer);
```

Comment:
@ydementieiev const?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)